### DISCUSSION TITLE
Openssl 3.3.2 download link broken

### DISCUSSION THREAD

**Discussion by SuzukiHonoka** [openssl/openssl#25377] [Q&A] [answered]
Broken link: https://www.openssl.org/source/openssl-3.3.2.tar.gz

Response:
```xml
<Error>
<Code>NoSuchKey</Code>
<Message>The specified key does not exist.</Message>
</Error>
```

**Comment by t8m** [comment, marked as answer]
We have changed the distribution method of the OpenSSL tarballs. We now use Github releases solely for this purpose.

Here are the official download links:
https://openssl-library.org/source/index.html
